src/pretix/static/pretixcontrol/js/ui/dragndroplist.js

```diff
@@ -1,48 +1,46 @@
 /*global $, Sortable*/
 $(function () {
     $("[data-dnd-url]").each(function(){
-        var container = $(this),
+        const container = $(this),
             url = container.data("dnd-url"),
             handle = $('<span class="btn btn-default btn-sm dnd-sort-handle"><i class="fa fa-arrows"></i></span>');
```

Comment:
I know, not part of this PR, but maybe just improve it to a real button?
```suggestion
            handle = $('<button type="button" class="btn btn-default btn-sm dnd-sort-handle"><i class="fa fa-arrows"></i></button>');
```

Reply:
lieber nicht, das verändert irgendwie das click-handling was die browser machen, jedenfalls wird mit einem `<button>` das drag-and-drop *manchmal* nicht ausgelöst, während es mit einem `<span>` zuverlässig ist